core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala

```diff
@@ -2159,6 +2171,16 @@ private[spark] class DAGScheduler(
     }
   }
 
+  private def isExecutorDecommissioned(bmAddress: BlockManagerId) = {
```

Comment:
If you don't mind, please add `taskScheduler` as a parameter here and use it inside function body.